lms/views/api/onedrive.py

```diff
@@ -0,0 +1,10 @@
+from pyramid.view import view_config
+
+
+@view_config(
+    request_method="GET",
+    route_name="onedrive.filepicker.authorize",
+    renderer="lms:templates/onedrive.html.jinja2",
+)
+def authorize(_request):
```

Comment:
```suggestion
def redirect_uri(_request):
```

I think the file should be `lms/views/onedrive.py` as I don't think this is an API endpoint

Reply:
I wonder if we could add a docstring explaining what this view is for? My understanding is that this is the URL that the OneDrive popup window redirects to and it must contain the `OneDrive.js` script that receives the redirect and shows the OneDrive file picker

Reply:
Added a bit of context in the docstring